### PR TITLE
Option to allow insecure connection

### PR DESCRIPTION
**Pull request by cail**
This gives an ability to skip all the funny games with certificates.
As soon as you don't care much somebody wants to steal your sensors )

### CODE REVIEW

File: src/InfluxDbClient.cpp [hunk 1]
```diff
@@ -221,6 +225,8 @@ bool InfluxDBClient::init() {
               wifiClientSec->setCACert(_certInfo);
          }
 #endif    
+        if (_insecure)
+            wifiClientSec->setInsecure();
```
Comment (vlastahajek) [inline src/InfluxDbClient.cpp:229]: `setInsecure` is not supported for ESP32. Move this to up to the conditioned compile part above.

File: src/InfluxDbClient.h [hunk 2]
```diff
@@ -242,6 +244,8 @@ class InfluxDBClient {
 #ifdef  ESP8266
   BearSSL::X509List *_cert = nullptr;   
 #endif
+    // if true - allow insecure connection
+    int _insecure = 0;
```
Comment (vlastahajek) [inline src/InfluxDbClient.h:248]: Why int? API param type is bool and it fits its purpose nice.

**Comment by vlastahajek** [comment]
@cail, thanks for the PR. I always wanted to have this capability in the lib. The issue is that ESP32 WifiSecureClient has different API and misses `setInsecure`. But as ESP32 doesn't verify certificates at all, we can have it as it is.

**Comment by cail** [comment]
fixed, thanks for comments.

**Pull request merged**